### ISSUE TITLE
Add a FLAC codec plugin

### ISSUE THREAD

**Issue by ctrlaltdavid** [vircadia/vircadia#1382]
This was raised as a possibility recently as potentially being a way to reduce audio bandwidth while not using much CPU. Add as a codec plugin alongside existing ones.

**Comment by JulianGro** [comment]
Doesn't FLAC use more bandwidth than Opus while also needing more CPU?
I guess Dale is going to try.

**Comment by daleglass** [comment]
No, actually FLAC uses a surprisingly small amount of CPU, at least in my testing. This makes it a contender for the niche that hifiAC used to fill, and it should perform better than zlib because it's targeted towards audio.

https://stsaz.github.io/fmedia/audio-formats/

**Comment by JulianGro** [comment]
Oh wow. That does look very promising.
